### PR TITLE
Ports CTRL+SHIFT Click from TG for admins

### PR DESCRIPTION
**Pull request by Xander3359**
## About The Pull Request
If you CTRL+SHIFT click yourself, it turns a ghost into a human.
## Why It's Good For The Game
Admin tools
## Changelog
:cl:
admin: CTRL+Shift click self to spawn as a human
/:cl:

### CODE REVIEW

**Comment by ivanmixo** [comment]
Why. We have a spess agent button

**Comment by Xander3359** [comment]
> Why. We have a spess agent button

Macro is a lot more convenient than a hidden button that clogs up UIs

**Comment by ivanmixo** [comment]
An obscure hotkey is surely a better solution. Make it make you a spatial agent or nothing.

**Comment by Xander3359** [comment]
>Make it make you a spatial agent or nothing.

If I do this, then when you CTRL SHIFT Click someone else's ghost, it will also make them a spatial agent.

**Comment by ivanmixo** [comment]
pointless

File: code/modules/mob/dead/observer/observer.dm [hunk 1]
```diff
@@ -964,3 +965,10 @@ GLOBAL_VAR_INIT(observer_default_invisibility, INVISIBILITY_OBSERVER)
 	if(!..())
 		return FALSE
 	visible_message(span_deadsay("[span_name("[src]")] points at [pointed_atom]."))
+
+/mob/dead/observer/CtrlShiftClickOn(mob/target_ghost)
+	if(src.ghost_vision && check_rights(R_SPAWN))
+		if(src == target_ghost)
+			src.client.holder.spatial_agent()
```
Comment (TiviPlus) [inline code/modules/mob/dead/observer/observer.dm:972]:
```suggestion
	if(ghost_vision && check_rights(R_SPAWN))
		if(src == target_ghost)
			client.holder.spatial_agent()
```

Comment (Xander3359) [reply code/modules/mob/dead/observer/observer.dm:972]: Forgot src is implied.

**Pull request merged**